drivers/usb/device/usb_dc_dw.c

```diff
@@ -653,7 +645,7 @@ static void usb_dw_isr_handler(void)
 			xfer_size = (grxstsp & USB_DW_GRXSTSR_PKT_CNT_MASK) >>
 			    USB_DW_GRXSTSR_PKT_CNT_OFFSET;
 
-			SYS_LOG_DBG("USB OUT EP%d: RX_FLVL status %d, size %d",
+			LOG_DBG("USB OUT EP%d: RX_FLVL status %d, size %d",
 				ep_idx, status, xfer_size);
```

Comment:
Does this line need adjustment?

Reply:
It also makes sense to add function name to the logs.

Reply:
Or, maybe, conversion LOG_DBG => USB_DBG might be done in the follow up patch, just fix adjustment.

Reply:
I added USB_* macros.